src/Nethermind/Nethermind.AccountAbstraction/Data/UserOperation.cs

```diff
@@ -37,6 +38,7 @@ public UserOperation(Address target, UInt256 nonce, byte[] callData, long callGa
             Signer = signer;
             Signature = signature;
             AccessList = accessList;
+            Hash = this.CalculateHash();
```

Comment:
its not going to be this easy

Reply:
Why so ?

Reply:
public static Keccak CalculateHash(this UserOperation userOperation) => Keccak.Compute(Rlp.Encode(userOperation).Bytes);
This won't work ?

Reply:
bc we're going to have to come up with a rlp serialization standard for user operations and implement it. take a look at Nethermind.Mev.Data.MevBundle.GetHash